### PR TITLE
QUARKUS-1070 Composable Platform

### PR DESCRIPTION
**Pull request by mjurc**
* Adding a test plan for QUARKUS-1070 Composable Platform streams
  and non-platform extension discovery.

### CODE REVIEW

File: QUARKUS-1070.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+There are unresolved topics with Dragonball release:
+* What should be included in sources?
+* Automated build hand-off from productization to participant QE teams.
+* Automated build sign-off by participant QE teams.
+
+## Contacts
+* Tester: Michal Jurč <mjurc@redhat.com>
+
+## References
+* Feature epic: [QUARKUS-1070 Composable Platform streams and non-platform extension discovery](https://issues.redhat.com/browse/QUARKUS-1070)
```
Comment (rsvoboda) [inline QUARKUS-1070.md:39]: * Internal registry: registry-proxy.engineering.redhat.com/rh-osbs/quarkus-mandrel-21-rhel8:21.3
 * Public registry: registry.access.redhat.com/quarkus/mandrel-21-rhel8:21.2

File: QUARKUS-1070.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+## Scope of the testing
+
+### Impact on test suites and testing automation
+* Red Hat build of Quarkus platform configuration has to be included in Maven repository tests
+* Quickstarts with RHBQ supported extensions have to be buildable and runnable with BOMs included in the Red Hat build 
+  of Quarkus platform releases
+* Code starter tests in start-stop test suite have to pass with code.quarkus portal updated with Red Hat build of 
+  Quarkus platform releases
+
```
Comment (rsvoboda) [inline QUARKUS-1070.md:21]: * Mandrel version considerations:
   * Initial platform release (RHBQ) needs to be tested against the internal build of Mandrel as Mandrel is usually released hand in hand with RHBQ
   * SP releases of the platform need to be tested with the public version of Mandrel as the internal registry can move ahead

File: QUARKUS-1070.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+For scalability, the release flow has to be automated and schedules of participant product releases need to be 
+consolidated.
+
+There are unresolved topics with Dragonball release:
+* What should be included in sources?
+* Automated build hand-off from productization to participant QE teams.
+* Automated build sign-off by participant QE teams.
+
+## Contacts
+* Tester: Michal Jurč <mjurc@redhat.com>
```
Comment (rsvoboda) [inline QUARKUS-1070.md:36]: Please add me too ;)

File: QUARKUS-1070.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+The goal of this epic is to
+
+* introduce a registry.quarkus.redhat.com for all devtools to consult for looking up platform and extensions
+* introduce bom generator in platform that generates uniform but composable BOMs that can be imported individually 
+  rather than all in bulk 
+* introduce notion of platform stream to indicate which minor version you are tracking and/or what you want to target 
+  when using tools
+
+## Scope of the testing
+
```
Comment (rsvoboda) [inline QUARKUS-1070.md:14]: Pls link "Starting point document for QE participants" https://docs.google.com/document/d/1qCOu4RiONV2PTTitJzJvRdnXYbUX2au5SXeF3ApupCI/edit
And mention that this is the initial plan from our side and that things are evolving ...

**Review by rsvoboda — CHANGES_REQUESTED** [review]

File: QUARKUS-1070.md (hunk 4 above)
Comment (mjurc) [reply QUARKUS-1070.md:14]: This is public repo, and we probably shouldn't show that link even in comments

File: QUARKUS-1070.md (hunk 1 above)
Comment (mjurc) [reply QUARKUS-1070.md:39]: Internal URLs shouldn't show on public repos :)

File: QUARKUS-1070.md (hunk 4 above)
Comment (rsvoboda) [reply QUARKUS-1070.md:14]: It's private document, so I don't think it's a big deal to have it even in comments

**Review by rsvoboda — APPROVED** [review]

**Pull request merged**